.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2870,7 +2870,15 @@ stages:
               --destination-path "$(Build.SourceVersion)" \
               --source "$(Build.ArtifactStagingDirectory)"
           displayName: Upload blobs to Azure
-          condition: and(succeeded(), eq(variables.isMainBranch, true))
+          condition: >
+            and(
+              succeeded(),
+              ne(variables['push_artifacts_to_azure_storage'], 'false'),
```

Comment:
Potentially a silly question, but is this line needed? I suppose it adds the ability to disable Azure pushes if one sets `push_artifacts_to_azure_storage == false`, but I haven't run into that before

Reply:
Yeah, agreed, I can't see a need for this line🤔

Reply:
I copy / pasted it from the s3 upload, which includes the following justification:
https://github.com/DataDog/dd-trace-dotnet/pull/3637/files#diff-2e09bd52f55cbb3965ceeae7f556dbbd458b94a4566d9631bea1204a4ad12198R2686

The false allows an explicit block, for the master branch. Might be useful at some point, but don't have strong opinion.